### DISCUSSION TITLE
Unable to include extensions in PrismaService

### DISCUSSION THREAD

**Discussion by RomanSkrypnik** [prisma/prisma#21520] [Q&A] [answered]
### Bug description

I'm using dependency injection to have an access to prisma instance: [Nest JS docs](https://docs.nestjs.com/recipes/prisma#use-prisma-client-in-your-nestjs-services)
Suddenly, I have noticed that it impossible to include extensions inside **PrismaService** class:

![image](https://github.com/prisma/prisma/assets/44501848/68d37e8d-3885-40f3-bf61-c65d11997869)
![image](https://github.com/prisma/prisma/assets/44501848/89f8717e-16fa-434d-bfa3-66ef7943dd3c)
![image](https://github.com/prisma/prisma/assets/44501848/97f0eb0d-a09b-4277-835a-1128f636674d)

Extension which I'm trying  to use: [prisma-extension-pagination](https://www.npmjs.com/package/prisma-extension-pagination)

### How to reproduce

Described in description

### Prisma information

Described in description

### Environment & setup

OS: MacOS 12.6
Database: PostgreSQL
Node.js version: 18.12.1

### Prisma Version

```
prisma                  : 4.15.0
@prisma/client          : 4.16.2
Current platform        : darwin
Query Engine (Node-API) : libquery-engine 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/@prisma/engines/migration-engine-darwin)
Format Wasm             : @prisma/prisma-fmt-wasm 4.15.0-28.8fbc245156db7124f997f4cecdd8d1219e360944
Default Engines Hash    : 8fbc245156db7124f997f4cecdd8d1219e360944
Studio                  : 0.484.0
Preview Features        : clientExtensions
```

**Comment by janpio** [comment]
What do you mean by "suddnely"? Did this use to work, and now does not work any more? What did you change in your app?

**Reply by RomanSkrypnik** [reply]
No, I just thought it's possible to do it that way
Speaking about changing code - everything is in "Bug description"

**Reply by RomanSkrypnik** [reply]
I don't see any example with dependency injection, only this way:
![image](https://github.com/prisma/prisma/assets/44501848/1a04becf-9437-4dba-b91b-0aec5ba5f197)

**Reply by RomanSkrypnik** [reply, marked as answer]
I have found this issue: https://github.com/prisma/prisma/issues/18628
